bin/node/src/chain_spec.rs

```diff
@@ -183,6 +184,17 @@ impl ChainParams {
     }
 }
 
+fn system_properties(token_symbol: String) -> serde_json::map::Map<String, Value> {
+    [
+        ("tokenSymbol".to_string(), Value::String(token_symbol)),
+        ("tokenDecimals".to_string(), Value::Number(Number::from(12))),
+        ("ss58Format".to_string(), Value::Number(Number::from(42))),
```

Comment:
Just a small nit, but maybe we could have token decimals and ss58 fromat defined in aleph_primitives?

Also I wonder if in future we would like to have more tokens apart from `AZERO` on mainnet or `TZERO` on testnet, because that would require changing this part. Probably not and this is an unnecessary concern

Reply:
token symbol can be passed form command line when generating chainspec

Reply:
Yes, it cam be passed from command line now, but chainspec supports multiple tokens at once. That's what i meant. But to be fair we don't need it right now

Reply:
yup, the client part of the config does not end up in the genesis block and can be updated later at will